net.longturn.freeciv21.yaml

```diff
@@ -0,0 +1,93 @@
+#   - flatpak-builder --install-deps-from=flathub --force-clean build/flatpak net.longturn.freeciv21.yaml
+#   - flatpak-builder --user --install --force-clean build/flatpak net.longturn.freeciv21.yaml
+#   - flatpak run net.longturn.freeciv21
+#   - flatpak remove net.longturn.freeciv21
+#
+##########
+
+app-id: net.longturn.freeciv21
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home
+  - --persist=.local/share/freeciv21
+  - --persist=.config/freeciv21
+
+rename-desktop-file: net.longturn.freeciv21.client.desktop
+rename-appdata-file: net.longturn.freeciv21.client.metainfo.xml
+
+command: freeciv21-client
+branch: stable
+default-branch: stable
+
+modules:
+  # Use Lua 5.3 shared-module as a dependency not included in the runtime.
+  - shared-modules/lua5.3/lua-5.3.5.json
+
+  # Get Libertunus font as a dependency.
+  - name: Libertinus
+    sources:
+      - type: archive
+        url: https://github.com/alerque/libertinus/releases/download/v7.040/Libertinus-7.040.zip
+        sha256: 2cce08507441d8ae7b835cfe51fb643ad5d9f6b44db4360c4e244f0e474a72f6
+        dest: fonts/Libertinus
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/share/freeciv21/fonts/Libertinus/static/OTF
+      - cp -rv fonts/Libertinus/static/OTF/*.otf ${FLATPAK_DEST}/share/freeciv21/fonts/Libertinus/static/OTF
+
+  # Now build and package freeciv21
+  - name: freeciv21
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DFREECIV_DOWNLOAD_FONTS=OFF
+    cleanup:
+      - /include
+      - /lib
+    post-install:
+      - install -D freeciv21-client.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install freeciv21-server.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.server.png
+      - install freeciv21-modpack.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.modpack.png
+      - desktop-file-edit --set-icon=${FLATPAK_ID} ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.client.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID}.server ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.server.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID}.modpack ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.modpack-qt.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID} ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.ruledit.desktop
+      - appstream-util modify ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.client.metainfo.xml id ${FLATPAK_ID}
+      - appstream-util modify ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.client.metainfo.xml launchable ${FLATPAK_ID}.desktop
+      - appstream-util modify ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.client.metainfo.xml release "3.0-stable"
```

Comment:
as I suspected this doesn't work.

Reply:
hm, there is a valid release tag in the file. What would that not validate?
```
<content_rating type="oars-1.0">
    <content_attribute id="social-chat">mild</content_attribute>
  </content_rating>
  <release>3.0-stable</release>
</component>
```

Reply:
Ok, I read the spec in more detail. I see what the issue is now.